src/NServiceBus.Persistence.DynamoDB/Config/DynamoDBClientProvidedByConfiguration.cs

```diff
@@ -2,8 +2,15 @@
 {
     using Amazon.DynamoDBv2;
 
-    class DynamoDBClientProvidedByConfiguration : IProvideDynamoDBClient
+    sealed class DynamoDBClientProvidedByConfiguration : IProvideDynamoDBClient
```

Comment:
It seems that with v8 we have embarked a bit on a journey of no longer using I in the interface to indicate "IDoSomething" but use now the more common "IDescribeSomething". Should we switch towards `IDynamoDBClientProvider` to align with that?

Reply:
👍

Reply:
Done